src/contracts/Permission/PermissionCheckerInterface.php

```diff
@@ -34,7 +34,21 @@ public function getContentCreateLimitations(Location $parentLocation): LookupLim
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
      */
-    public function getContentUpdateLimitations(Location $parentLocation): LookupLimitationResult;
+    public function getContentUpdateLimitations(Location $location): LookupLimitationResult;
```

Comment:
What is the reason for changing existing contract parameter name?

Reply:
Because it didn't match the parameter name from one of the interface's implementations and it did not make much sense to me. We can look at the original PR you will see that both names did not match each other back then: 
- interface: https://github.com/ezsystems/ezplatform-admin-ui/pull/1273/files#diff-14fc449f7ecdcea5e8c83c4f5be3656dfceee1ed2df4eb40cea59c2e9ae36d16R37
- implementation: https://github.com/ezsystems/ezplatform-admin-ui/pull/1273/files#diff-c3310f3f33562f15c00a3cfdee25ffbd7f0e88db43c1dea1fe87001dd0ecb785R211

It was set to `parentLocation` in 4.6 here: https://github.com/ibexa/admin-ui/pull/872/files#diff-d1039d1a73d7e40070b874e926dbb079bee41c510c4b14ef555ba1d8f7a66f20R32

But I think this should have been `location` as done in the initial method implementation: https://github.com/ezsystems/ezplatform-admin-ui/pull/1273/files#diff-c3310f3f33562f15c00a3cfdee25ffbd7f0e88db43c1dea1fe87001dd0ecb785R211.

Reply:
I know, I think last time we tentatively agreed that we are not going to have BC on parameters naming.